### DISCUSSION TITLE
Continuous Clear

### DISCUSSION THREAD

**Discussion by gentoolman** [LmeSzinc/AzurLaneAutoScript#3748] [Q&A]
How does the Continuous Clear Option work? I tried several Input Types - e.g. 4-1, 4-2, 4-3... or 4-1 to 6-4 or 4-1;4-2;4-3... or just with spaces. nothing seems to work

INFO     21:56:59.166 │ Loading OCR dependencies                                   
WARNING  21:56:59.166 │ Map file not found:                                        
         campaign.campaign_main.campaign_4_1to6_4                                  
.
.
.
.
CRITICAL 21:56:59.170 │ Possible reason #1: This event (campaign_main) does not    
         have campaign_4_1to6_4                                                    
CRITICAL 21:56:59.170 │ Possible reason #2: You are using an old Alas, please check
         for update, or make map files yourself using dev_tools/map_extractor.py   
CRITICAL 21:56:59.171 │ Request human takeover                                     
INFO     21:56:59.651 │ No provider specified, skip sending

**Comment by LmeSzinc** [comment]
type `4-1`, stage names will be auto increased
